src/graph/sampling/randomwalks/randomwalks.cc

```diff
@@ -93,7 +93,7 @@ std::tuple<IdArray, IdArray, TypeArray> RandomWalkWithStepwiseRestart(
 
   TypeArray vtypes;
   std::pair<IdArray, IdArray> result;
-  ATEN_XPU_SWITCH(hg->Context().device_type, XPU, "RandomWalkWithStepwiseRestart", {
+  ATEN_XPU_SWITCH_CUDA(hg->Context().device_type, XPU, "RandomWalkWithStepwiseRestart", {
```

Comment:
Could you also enable CUDA support for `RandomWalk` and `RandomWalkWithRestart`?  The former could be done with skipping the `restart_prob` logic in `RandomWalkWithStepwiseRestart` and the latter could be done with an array with the same value I think.